runtime/src/main/java/org/corfudb/runtime/ViewsMaintenance.java

```diff
@@ -23,39 +24,71 @@
 
 @Slf4j
 @NotThreadSafe
-public class ViewsGarbageCollector {
+public class ViewsMaintenance {
 
 
     private long trimMark = Address.NON_ADDRESS;
 
     @Getter
     private boolean started;
 
-    final ScheduledExecutorService gcThread = Executors.newSingleThreadScheduledExecutor(
+    @Getter
+    @Setter
+    long objectSyncDelayInSeconds = 30;
+
+    @Getter
+    @Setter
+    long objectGCPeriodInSeconds = 20 * 60;
+
+    final ScheduledExecutorService maintenance = Executors.newSingleThreadScheduledExecutor(
```

Comment:
hopefully running both tasks on the same single threaded executor is ok. Could you just please add a comment.

Reply:
I think it should be OK. GC is just purely in memory operation and if object syncing takes soo long that the in memory GC becomes a problem, then the sync it self will probably be an issue before the side effects of not doing GC becomes an issue.